apps/rocketchat/template-rocketchat-mongodb.yaml

```diff
@@ -350,7 +383,7 @@ objects:
       name: "${MONGODB_SERVICE_NAME}"
     spec:
       serviceName: "${MONGODB_SERVICE_NAME}-internal"
-      replicas: "${MONGODB_REPLICAS}"
+      replicas: ${{MONGODB_REPLICAS}}
```

Comment:
Why the double brackets?

Reply:
string for replicas doesn't work for me, have to be an integer. Is there another way to fix this?